pubkey.pem

```diff
@@ -0,0 +1,4 @@
+-----BEGIN PUBLIC KEY-----
```

Comment:
I doubt we need this file, what is it for?

Reply:
oh, I missed it